cumulus/ansible/tasks/playbooks/default.yml

```diff
@@ -0,0 +1,39 @@
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  vars:
+    cluster_state: running
+    cluster_region: us-west-2
+    instance_type: t2.medium
+    ec2_pod_spec:
+      key: "{{ aws_keyname }}"
+      cluster_name: "{{ cluster_id }}"
+      rules:
+
+        - proto: tcp # ssh
+          from_port: 22
+          to_port: 22
+          cidr_ip: 0.0.0.0/0
+
+        - proto: all
+          group_name: "ec2_pod_{{cluster_id}}"
+          group_desc: "Ec2 security group for cluster {{ cluster_id }}"
+
+      hosts:
+        head:
+          type: "{{ instance_type }}"
+
+        data:
+          type: "{{ instance_type }}"
+          count: "{{ instance_count }}"
+  roles:
```

Comment:
We will need to be able to specify different instance types for the head and data nodes.